### ISSUE TITLE
Improve Spotify playlist generation

### ISSUE THREAD

**Issue by jamesdabbs** [jamesdabbs/ticketron#1]
* Want more songs in playlist from artists playing shows sooner (`(180 - days_til_show) / 18`?)
* Nice to have: be able to add more than 10 tracks, but that looks to require integration w/ EchoNest (see https://developer.spotify.com/web-api/get-artists-top-tracks)
* Nice to have: user control panel for customizing name of playlist, max tracks per artist, number of day horizon.

**Comment by jamesdabbs** [comment]
Once this is done, you should be able to run `rake playlist:generate` and see the resultant playlist on Spotify.
